### PR TITLE
Update hosts

### PR DESCRIPTION
**Pull request by vvarg229**
Script to modify IP addresses in the hosts file, the CoreDNS file and the Morph Chain config

### CODE REVIEW

**Review by cthulhu-rider — APPROVED** [review]
tried, works fine! @vvarg229 is it worth mentioning this script in some doc? maybe it will help someone (`sudo` to modify `/etc/hosts` worth a word of caution)

**Comment by vvarg229** [comment]
> tried, works fine! @vvarg229 is it worth mentioning this script in some doc? maybe it will help someone (`sudo` to modify `/etc/hosts` worth a word of caution)

Yes, that's a great idea! Thanks so much for the review and the idea!
I'll add it to the readme file.

**Review by roman-khimov — COMMENTED** [review]
`update_hosts.sh` should reside in `bin`, and yes, we need some documentation for it.

It looks dangerous at the same time, I'd add.

**Comment by vvarg229** [comment]
> tried, works fine! @vvarg229 is it worth mentioning this script in some doc? maybe it will help someone (`sudo` to modify `/etc/hosts` worth a word of caution)

I added documentation and tests

**Comment by vvarg229** [comment]
> `update_hosts.sh` should reside in `bin`, and yes, we need some documentation for it.
> 
> It looks dangerous at the same time, I'd add.

I added documentation and tests.
Tests to make it not look so dangerous:)

File: bin/update_hosts.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+: "${COREFILE:=services/coredns/Corefile}"
+: "${MORPH_CHAIN_CONFIG:=services/morph_chain/protocol.privnet.yml}"
+temp_file=$(mktemp)
+
+make hosts > "$temp_file"
+
+morph_chain_ip=$(grep "morph-chain.neofs.devenv" "$temp_file" | awk '{print $1}')
+
+sed -i -E "s/(nns[[:space:]]*http\:\/\/)([0-9]{1,3}[.]){3}[0-9]{1,3}(:30333)/\1$morph_chain_ip\3/" "$COREFILE"
+sed -i -E "s/Address:[[:space:]]+([0-9]{1,3}[.]){3}[0-9]{1,3}/Address: $morph_chain_ip/" "$MORPH_CHAIN_CONFIG"
```
Comment (roman-khimov) [inline bin/update_hosts.sh:14]: Likely needs to be updated after #249.

File: README.md [hunk 2]
```diff
@@ -155,6 +155,24 @@ Clean up `vendor` directory.
 
 ## How to prepare environment to run neofs-testcases
 
+If necessary, change the IPV4_PREFIX value in the .env file from 192.168.130
+to the existing subnet on the host.
+Importantly, the subnet mask must be 24.
+Use ```ip a``` tosee the network configuration on the host.
```
Comment (cthulhu-rider) [inline README.md:161]: small typo

**Review by cthulhu-rider — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (vvarg229) [reply README.md]: Done

File: bin/update_hosts.sh (hunk 1 above)
Comment (vvarg229) [reply bin/update_hosts.sh]: Done

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**